code/modules/maps/reader.dm

```diff
@@ -3,7 +3,7 @@
 //////////////////////////////////////////////////////////////
 
 //global datum that will preload variables on atoms instanciation
-var/global/use_preloader = FALSE
+GLOBAL_VAR_INIT(use_preloader, FALSE)
 var/global/dmm_suite/preloader/_preloader = new
```

Comment:
this should be GLOB too

Reply:
If it doesn't prevent the MC or round from working, out of scope

Reply:
You touched the file, just change it, it's 1 line